integration/okhttp3/src/main/java/com/bumptech/glide/integration/okhttp3/OkHttpUrlLoader.java

```diff
@@ -0,0 +1,83 @@
+package com.bumptech.glide.integration.okhttp3;
+
+import android.content.Context;
+
+import com.bumptech.glide.load.Options;
+import com.bumptech.glide.load.model.GlideUrl;
+import com.bumptech.glide.load.model.ModelLoader;
+import com.bumptech.glide.load.model.ModelLoaderFactory;
+import com.bumptech.glide.load.model.MultiModelLoaderFactory;
+
+import java.io.InputStream;
+
```

Comment:
Build failed with:
> integration/okhttp3/OkHttpUrlLoader.java:13: error: Wrong order for 'okhttp3.Call' import.

Try `gradlew clean check jar assemble` before commit, it may be a little overkill, but it should check most things if they broke.